src/milind/PrimeNumberAndAdditionAndAverage.java

```diff
@@ -0,0 +1,34 @@
+package milind;
+
+import java.util.Arrays;
+
+public class PrimeNumberAndAdditionAndAverage {
+	
+	void primeNumber(int startRange, int endRange) {
+		int sum = 0;
+		int countPrimeNumber = 0;
+		for(int index = startRange; index <= endRange; index++) {
+			int count = 0;
+			for(int innerIndex = startRange; innerIndex <= index; innerIndex++) {
+				if(index % innerIndex == 0) {
+					count++;
+				}

```

Comment:
once you know this number will not be prime number then you can put break in if condition.